src/http.js

```diff
@@ -214,7 +221,7 @@ function buildFormData(reqForm) {
    * @return {FormData} - new FormData instance
    */
   return Object.entries(reqForm).reduce((formData, [name, input]) => {
-    if (!input.collectionFormat && !input.isOAS3formatArray) {
+    if (!(input.collectionFormat && input.collectionFormat === 'multi') && !input.isOAS3formatArray) {
```

Comment:
More readable suggestion
```suggestion
    if (has(input, 'collectionFormat') && input.collectionFormat !== 'multi' && !input.isOAS3formatArray) {
```
[has](https://lodash.com/docs/4.17.15#has) function comens from `lodash`

Reply:
I think my suggestion is logically wrong because I misunderstood the negated conjunction.